lib/tasks/book2.rake

```diff
@@ -184,42 +184,18 @@ end
 
 desc "Generate book html directly from git repo"
 task remote_genbook2: :environment do
-  @octokit = Octokit::Client.new(login: ENV["API_USER"], password: ENV["API_PASS"])
-  all_books = {
-    "az" => "progit2-aze/progit2",
-    "be" => "progit/progit2-be",
-    "bg" => "progit/progit2-bg",
-    "cs" => "progit-cs/progit2-cs",
-    "de" => "progit/progit2-de",
-    "en" => "progit/progit2",
-    "es" => "progit/progit2-es",
-    "fr" => "progit/progit2-fr",
-    "gr" => "progit2-gr/progit2",
-    "id" => "progit/progit2-id",
-    "it" => "progit/progit2-it",
-    "ja" => "progit/progit2-ja",
-    "ko" => "progit/progit2-ko",
-    "mk" => "progit2-mk/progit2",
-    "nl" => "progit/progit2-nl",
-    "pl" => "progit2-pl/progit2-pl",
-    "pt-br" => "progit/progit2-pt-br",
-    "pt-pt" => "progit2-pt-pt/progit2",
-    "ru" => "progit/progit2-ru",
-    "sl" => "progit/progit2-sl",
-    "sr" => "progit/progit2-sr",
-    "tl" => "progit2-tl/progit2",
-    "tr" => "progit/progit2-tr",
-    "uk" => "progit/progit2-uk",
-    "uz" => "progit/progit2-uz",
-    "zh" => "progit/progit2-zh",
-    "zh-tw" => "progit/progit2-zh-tw",
-    "fa" => "progit2-fa/progit2"
-  }
+  Octokit.auto_paginate = true
+  if ENV["GITHUB_API_TOKEN"]
+    @octokit = Octokit::Client.new(access_token: ENV["GITHUB_API_TOKEN"])
+  else
+    @octokit = Octokit::Client.new(login: ENV["API_USER"], password: ENV["API_PASS"])
+  end
```

Comment:
I understand why the list of repositories had to be moved. But these Octokit changes seem unrelated. I _think_ they're probably a good thing, though (and it looks like they match the way the manpages set it up).

Reply:
Oops, sorry. I changed it to work with my personal setup and it slipped through in the commit.

Reply:
I think the end result is OK, but I would generally prefer to see it in a separate commit. :)